Modules/Filtering/ImageIntensity/include/itkRoundImageFilter.h

```diff
@@ -47,7 +48,18 @@ class Round
   inline TOutput
   operator()(const TInput & A) const
   {
-    return itk::Math::Round<TOutput, TInput>(A);
```

Comment:
Isn't it safer to leave as is? IMHO, it would be nice to make Round work with floating output, but maybe not this time?